page-changelog.php

```diff
@@ -13,7 +13,117 @@
+
+<a name="latest8.2"></a></li>
+<h3>Version 8.2.9 <small>Oct 26 2016</small></h3></li>
+<ul>
+<li>Core: Adjusted documentation link to issue template - <a href="https://github.com/owncloud/core/issues/26087">core/#26087</a></li>
+<li>Core: Display feedback in users page when changing password - <a href="https://github.com/owncloud/core/issues/25532">core/#25532</a></li>
+<li>Core: Fix mime type detection in hidden directories - <a href="https://github.com/owncloud/core/issues/26133">core/#26133</a></li>
+<li>Core: Change forum URL to central - <a href="https://github.com/owncloud/core/issues/25644">core/#25644</a></li>
+<li>Core: Fix unshare from self etag propagation - <a href="https://github.com/owncloud/core/issues/26346">core/#26346</a></li>
+<li>Updater: Fix signature data not found error when updating to 9.0 - <a href="https://github.com/owncloud/updater/issues/384">updater/#384</a></li>
+<li>Updater: Clicking notification now updates doc in new tab - <a href="https://github.com/owncloud/core/issues/26260">core/#26260</a></li>
+<li>User_external: Double verify the SMB response - <a href="https://github.com/owncloud/apps/issues/2198">apps/#2198</a></li>
+</ul>
+Download: <a href="https://download.owncloud.org/community/owncloud-8.2.9.tar.bz2">owncloud-8.2.9.tar.bz2</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.9.zip">owncloud-8.2.9.zip</a></br>
+MD5: <a href="https://download.owncloud.org/community/owncloud-8.2.9.tar.bz2.md5">owncloud-8.2.9.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.9.zip.md5">owncloud-8.2.9.zip.md5</a></br>
+SHA256: <a href="https://download.owncloud.org/community/owncloud-8.2.9.tar.bz2.sha256">owncloud-8.2.9.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.9.zip.sha256">owncloud-8.2.9.zip.sha256</a></br>
+PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-8.2.9.tar.bz2.asc">owncloud-8.2.9.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.9.zip.asc">owncloud-8.2.9.zip.asc</a></br>
+Packages on our Build Service: <a href="https://download.owncloud.org/download/repositories/8.2/owncloud">8.2 Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
+
+<a name="latest8.1"></a></li>
+<h3>Version 8.1.11 <small>Oct 26 2016</small></h3></li>
+<ul>
+<li>Core: Adjusted documentation link to issue template - <a href="https://github.com/owncloud/core/issues/26087">core/#26087</a></li>
+<li>Updater: Fixed to properly use update server URL from config - <a href="https://github.com/owncloud/core/issues/381">core/#381</a></li>
```

Comment:
I hope one day I'll find a way to auto-generated these changelogs instead of typing by hand and making such mistakes... Or alternatively have a bot that pre-checks all links.